desk/app/contacts.hoon

```diff
@@ -238,11 +238,10 @@
     ::
     ++  p-init
       |=  wen=(unit @da)
+      ::  TODO: figure out if we can just give the profile every time
```

Comment:
Not sure why this was set up this way, because the payload is Small Enough™. But giving it _every time_ regardless of the timestamp in the subscription is slightly more expensive, data transfer-wise, so I wouldn't strip that functionality out.